lib/checks/aria/valid-scrollable-semantics.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "valid-scrollable-semantics",
```

Comment:
I don't understand why this is a separate check. It seems to me this is overcomplicating things.

Reply:
i may not understand the suggestion.

i'd make the same argument with respect to checks being available for use in future rules. It seems nice to have has-widget-role as a check, and this is the other case for which we would PASS elements as having valid semantics for being in the focus order.

How else would we want to express that, other than having this check?